crosshair/core.py

```diff
@@ -653,6 +651,8 @@ def split_limits(
         self, priority: float
     ) -> Tuple["AnalysisOptions", "AnalysisOptions"]:
         """
+        Split the max iterations of the analysis in two.
```

Comment:
```suggestion
        Divide resource allotments (timeouts, iteration caps) into allotments for two stages of analysis.
```

Reply:
This will be too long for the first summary sentence.

What about:
```
Divide resource allotments into two.

The resource allotments (timeouts, iteration caps) are split into allotments for two stages of analysis.
```
?

Reply:
I made a commit changing the docstring so that you can examine how it looks in the "real".